### PR TITLE
reinitalizing bookmarks model+creating migrations files

### LINKED ISSUE #21: Favorite link list-Data model classes
Create the model classes that describe the data model of the application.

### LINKED ISSUE #22: Favorite link list- Migration files for initializing the data model
We need to create the database table that stores the data it represents for this feature. In Django, we do it via “database migrations”. Database migrations are scripts that can create or alter the database structure.

### LINKED ISSUE #23: Favorite link list- Migration files for creating test data for your DEV environment
Create a custom database migration file that will automatically add some test data for us to work with.

### LINKED ISSUE #27: Favorite link list- Data model functions
Create a personalized tool for a student with these functionalities:
- Add, remove and rename links
- Tag link as favorite
- Show frequently used links by date

### LINKED ISSUE #39: setting Django admin UI
In order to present our data model

### PR DESCRIPTION
**Pull request by noaramon**
This PR is an update to a our data model due to comments from our reviewers and contains admin.py file in order to link and show our project in Django admin UI.
Fixes #21 
Fixes #22 
Fixes #23 
Fixes #27 
Fixes #39

### CODE REVIEW

File: StuDash/settings.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "main",
     "accounts",
     "forumMessages",
-    "forumMessages.apps.ForumMessagesConfig",
```
Comment (Omeramsc) [inline StuDash/settings.py]: note that this is also a part of #49

**Review by Omeramsc — CHANGES_REQUESTED** [review]
what the empty `test` and `test_data.py` files are for? 

also - please link an issue to the PR.

**Comment by ravidbro** [comment]
I find it strange that you have a Pipfile.lock without Pipfile in the same PR.
Please rebase on 'main' and see if you still have changes in this file and if you need them

**Comment by noaramon** [comment]
> I find it strange that you have a Pipfile.lock without Pipfile in the same PR.
> Please rebase on 'main' and see if you still have changes in this file and if you need them

We had to make changes in Pipfile.lock in order to insert 'bootstrap4' which is an CSS library.

**Comment by ravidbro** [comment]
1. Now you have a conflict on settings.py that needs to be resolved.
2. I thought that a bookmark should have a relation to 'Course' table.

File: bookmarks/models.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+class Bookmark (models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    url = models.URLField(max_length=200)
+    urlname = models.CharField(max_length=30)
+    last_view_date = models.DateTimeField(auto_now=True, null=True)
+
+    @classmethod
+    def addbookmark(cls, current_user, new_link):
+        new_link.user.add(current_user)
```
Comment (ravidbro) [inline bookmarks/models.py]: Since 'user' is not a list, I don't think that add is the right method to use here.

File: bookmarks/models.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    @classmethod
+    def removebookmark(cls, current_user, remove_link):
+        remove_link.user.remove(current_user)
```
Comment (ravidbro) [inline bookmarks/models.py]: same, 'user' is not a list, so not 'remove'

**Review by ravidbro — CHANGES_REQUESTED** [review]

File: bookmarks/models.py (hunk 2 above)
Comment (noaramon) [reply bookmarks/models.py]: Deleted this method

File: bookmarks/models.py (hunk 3 above)
Comment (noaramon) [reply bookmarks/models.py]: Deleted this method

**Review by ravidbro — APPROVED** [review]

**Review by Omeramsc — APPROVED** [review]
LGTM, good job!

**Pull request merged**